src/app.css

```
@@ -152,3 +152,55 @@ pre {
 		animation: none !important;
 	}
 }
+
+/* ── Touch target fixes ───────────────────────────────────────────── */
+
+/* Increase touch target sizes for mobile - WCAG 2.5.5 Target Size (AAA) */
+@media (hover: none) and (pointer: coarse) {
+	/* btn-xs: 24px → 44px minimum */
+	.btn-xs:not(.btn-square, .btn-circle) {
+		--size: calc(var(--size-field, 0.25rem) * 11);
+		min-height: 2.75rem;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)